### ISSUE TITLE
New user script fails on authentication to mongo

### ISSUE THREAD

**Issue by jackconnorrichards** [lsumedia/lcr-web#7]
Mongoose connection in newuser.js needs modifying to pass the user/password details from the config.json file

**Comment by jdaandersj** [comment]
This is because the create user script expects mongo to not be running with authentication. 

I am probably going to make lcr-web operate through docker-compose with a dedicated and unexposed MongoDB container. Therefore, authentication to MongoDB will eventually not be necessary.

Will close this once docker-compose is working.

**Comment by camerontsmall** [comment]
I created a Docker config and updated a lot of the backend already on the
docker-reorganisation branch, might be helpful for you. I ran into some
issues with getting TypeScript working with mongoose though, actually got a
bit stuck with that. Depending on how comfortable you are with TS you could
take the frontend containers from that and just revert the backend one to
the current JavaScript version

**Comment by jdaandersj** [comment]
I am not particularly comfortable with TS so might just have a look at your branch and then reorganise what is already working. Not sure how I had completely missed your docker branch 🤦

**Comment by camerontsmall** [comment]
Yeah maybe one step at a time! It's a shame because TypeScript is super helpful for making sure data types are the same between backend & frontend. We've started looking into https://docs.nestjs.com/  at work recently, wonder if it could come in handy for this project in the future
